app/views/authors/new.html.erb

```diff
@@ -1,12 +1,25 @@
-<%= form_tag '/authors' do %>
-  <%=label_tag 'first_name' %>
-  <%=text_field_tag 'first_name' %>
+<div>
+  <% if flash[:success] %>
```

Comment:
Wouldn't rails handle all this newly added staff if you use `form_for` helper instead of `form_tag`? I realize @author is not an instance of ApplicationRecord but it seems including some additional modules in Author class would save us from this manual processing.